### DISCUSSION TITLE
reload bootloader?

### DISCUSSION THREAD

**Discussion by ror1948** [EdgeTX/edgetx#3142] [Q&A] [answered]
when does one have to reload bootloader? is it required to do on nighty builds?

**Comment by pfeerick** [comment, marked as answer]
No. At most, at every major version, but even then it's not strictly necessary. The bootloader is not something that is often touched. If there was ever a definite reason to update it, it would be explicitly mentioned in the release notes.

**Comment by ror1948** [comment]
Ok thanks
